deluge/ui/console/modes/preferences/preference_panes.py

```diff
@@ -300,7 +300,7 @@ def create_pane(self, core_conf, console_config):
         self.add_header(_('Outgoing Interface'), space_above=True)
         self.add_text_input(
             'outgoing_interface',
-            _('IP address of the interface to open outgoing connections on. (leave empty for default):'),
+            _('The interface adapter name to open outgoing connections on. (leave empty for default):'),
```

Comment:
Remove `The`

Reply:
Interface adapter name for outgoing BitTorrent connections. (Leave  empty for default).